### ISSUE TITLE
Is this project still OK?

### ISSUE THREAD

**Issue by danr777** [ryandoyle/vagrancy#9]
Is this still a relevant project? I have set it up but its not accepting .box files when I try to upload to the puma server using curl on OSX

Is there an alternative to privately hosting .box files?

**Comment by witjoh** [comment]
installed it last week.  Have no problems uploading boxes.

**Comment by danr777** [comment]
Thanks for responding!... yeah I am   doing a:

curl --progress-bar http://myhost.com:8099/test/testdev/1.0.0/virtualbox/ --upload-file devtest.box -v 

getting this on the console:

* TCP_NODELAY set
* Connected to myhost.com (10.77.78.97) port 8099 (#0)
> PUT /test/testdev/1.0.0/virtualbox/devtest.box HTTP/1.1
> Host: myhost.com:8099
> User-Agent: curl/7.54.0
> Accept: */*
> Content-Length: 6198087229
> Expect: 100-continue
> 
* Done waiting for 100-continue

and it just sits there.... never see anything on the host console of the vagrancy status window... if i browse to it I see the normal GET activity...

not sure what it is.. I've given full permissions on the folders to....   it auto creates the folders on upload right?

**Comment by witjoh** [comment]
Yes it creates the folders.  The image is first stored in the /tmp folder.  I noticed the cleanup of those files is not done after the upload is done. When /tmp is full, I got a connection rest and needed to cleanup the /tmp directory.  I'm running on rhel 7.4
